x-pack/plugins/enterprise_search/public/applications/analytics/components/analytics_collection_view/analytics_collection_integrate.tsx

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import {
+  EuiCodeBlock,
+  EuiDescriptionList,
+  EuiPanel,
+  EuiSpacer,
+  EuiText,
+  EuiTitle,
+} from '@elastic/eui';
+
+import { i18n } from '@kbn/i18n';
+
+import { AnalyticsCollection } from '../../../../../common/types/analytics';
+import { getEnterpriseSearchUrl } from '../../../shared/enterprise_search_url';
+
+interface AnalyticsCollectionIntegrateProps {
+  collection: AnalyticsCollection;
+}
+
+export const AnalyticsCollectionIntegrate: React.FC<AnalyticsCollectionIntegrateProps> = ({
+  collection,
+}) => {
+  const analyticsDNSUrl = getEnterpriseSearchUrl(`/analytics/${collection.name}`) as string;
+  const credentials = [
+    {
+      title: i18n.translate(
+        'xpack.enterpriseSearch.analytics.collections.integrate.credentials.collectionName',
```

Comment:
suggestion: include `collectionsView` and `tab` into the localisation string

For example: `xpack.enterpriseSearch.analytics.collections.collectionsView.integrateTab`